src/main/java/tracko/commons/core/Messages.java

```diff
@@ -8,6 +8,7 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_ORDERS_FOUND_OVERVIEW = "%1$d orders found!";
+    public static final String MESSAGE_ONE_ORDER_FOUND = "1 order found!";
```

Comment:
Can rename to just MESSAGE_ORDERS_FOUND = "%1$d order(s) found!", and change the logic as such to make the general code shorter and easier to test.